llama_parse/base.py

```diff
@@ -303,6 +303,25 @@ def validate_base_url(cls, v: str) -> str:
         url = os.getenv("LLAMA_CLOUD_BASE_URL", None)
         return url or v or DEFAULT_BASE_URL
 
+    _aclient: Union[httpx.AsyncClient, None] = PrivateAttr(default=None, init=False)
+
+    @property
+    def aclient(self) -> httpx.AsyncClient:
```

Comment:
this is actually done wrong. should have been setting an instance attribute rather than just creating a client. this is remedied in the second commit.